lib/vsc/install/commontest.py

```diff
@@ -87,15 +93,16 @@ class CommonTest(TestCase):
     #   Whitelist: if match, fail test
     PROSPECTOR_BLACKLIST = [
         # 'wrong-import-position',  # not sure about this, these usually have a good reason
+        "Redefining built-in 'basestring'",  # basestring is defined when running on top of Python 3.x
```

Comment:
actually, you should remove this one too. just ack the pylint failure on the 2 lines  in vsc-install where you do the redefinition.
for all other repos that use this, it's better to introduce a `is_string` function in `vsc.utils.missing`, and in that module, also ack the redefinition (or better yet, make a function that doesn't require a redefinition). i'm going to assume that most use cases of basestring are from `isinstance(somestring, basestring)`, so that test should be modified everywhere (it's more work, but much cleaner)